src/main/java/com/google/sps/model/Trip.java

```diff
@@ -1,18 +1,22 @@
 package com.google.sps.model;
 
 import com.google.auto.value.AutoValue;
-import java.util.List;
+
+import java.util.HashSet;
+import java.util.TreeSet;
+import java.util.Set;
 
 @AutoValue
 public abstract class Trip {
 
     static Trip create(long tripID, String title, float totalBudget){
-        return new AutoValue_Trip(tripID, title, totalBudget);
+        return new AutoValue_Trip(tripID, title, new HashSet<Long>(), new TreeSet<Long>(), 0.0F, totalBudget);
```

Comment:
Why not a HashSet for both?

Reply:
Good point. I was thinking that events should be ordered chronologically but I'm just now realizing that the set only holds the ids.